packages/reports/src/reports/editors/simple/SelectorUI.jsx

```diff
@@ -107,22 +164,82 @@ function SelectorUI(props) {
       onChange={setSelectorType}
       data={Object.values(SELECTOR_TYPE)}
     />
+    <Select
+      label="Options Type"
+      value={optionsType}
+      onChange={setOptionsType}
+      data={Object.values(OPTION_TYPE)}
+    />
+    {
+      optionsType === OPTION_TYPE.STATIC
+        ? <StaticSelectorOptionsEditor
+          staticOptions={options.static}
+          setStaticOptions={optionsState => setOptions({...options, static: optionsState})}
+        />
+        : <DynamicSelectorOptionsEditor
+          dynamicOptions={options.dynamic}
+          setDynamicOptions={optionsState => setOptions({...options, dynamic: optionsState})}
+          id={id}
+        />
```

Comment:
I wonder if we could develop some sort of standard for conditionally rendering components like this. I have seen both:

- Declare the comps in a lookup object, before the render, and pull one out and insert it as `{myComponent}`.
- Declare the lookup object INLINE. Dave did this here once:
![image](https://user-images.githubusercontent.com/1714292/152230317-46ae01f1-941c-4074-8202-99ece5da8950.png)

Not sure which one I like. Just food for thought.